src/vim.js

```diff
@@ -4899,40 +4923,50 @@ export function initVim(CodeMirror) {
   /** @arg {string} str */
   function translateRegex(str) {
     // When these match, add a '\' if unescaped or remove one if escaped.
+    var modes = {
+      V: '|(){+?*.[$^', // verynomagic
+      M: '|(){+?*.[', // nomagic
+      m: '|(){+?', // magic
+      v: '<>', // verymagic
+    };
+    var escapes = {
+      '>': '(?<=[\\w])(?=[^\\w]|$)',
+      '<': '(?<=[^\\w]|^)(?=[\\w])',
+    };
+    // TODO the default is supposed to be modes.m but the old implementation missed +?
     var specials = '|(){';
```

Comment:
any reason not to update it?

```suggestion
    var specials = modes.m;
```

?